content/en/blog/k8-basics/index.md

```diff
@@ -0,0 +1,212 @@
+---
+title: "K8 Basics"
```

Comment:
k8 references should be changed to kubernetes or k8s basics

Reply:
Sure, I will change it to k8s